### PR TITLE
Fix/fix bugs validation

### PR DESCRIPTION
**Pull request by Rey-2903**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "configurations": [
+        {
+            "type": "chrome",
+            "request": "launch",
+            "name": "Launch Chrome against localhost",
+            "url": "http://localhost:3000",
+            "webRoot": "${workspaceFolder}"
+        }
+    ]
+}
```
Comment (zavarin-michael) [inline .vscode/launch.json:15]: explain?
i would not add this file in repo

File: components/CirclesComponents/CircleCreateForm.tsx [hunk 2]
```diff
@@ -97,10 +105,43 @@ export default function CircleCreateForm() {
...
+                    rules={[
+                        {
+                            validator: (_, value) => {
+                                if (/^[0-9]+$/.test(value)) {
+                                    return Promise.resolve();
+                                } else {
+                                    return Promise.reject("Введите цифры");
+                                }
+                            },
+                            whitespace: true,
```
Comment (zavarin-michael) [inline components/CirclesComponents/CircleCreateForm.tsx:121]: should it be false?
i think user shouldn't eneter any whitespace in this input

File: components/EmployeeComponents/AddEmployee.tsx [hunk 3]
```diff
@@ -93,7 +95,14 @@ export default function AddEmployee() {
                         rules={[
                             {
                                 required: true,
-                                message: "Пожалуйста введите номер телефона",
+                                whitespace: true,
```
Comment (zavarin-michael) [inline components/EmployeeComponents/AddEmployee.tsx:98]: ^^

**Review by zavarin-michael — CHANGES_REQUESTED** [review]

File: components/CirclesComponents/CircleCreateForm.tsx (hunk 2 above)
Comment (Rey-2903) [reply components/CirclesComponents/CircleCreateForm.tsx:121]: The white space parameter in the antd library is responsible for checking for entering spaces, so it has the value true

File: .vscode/launch.json (hunk 1 above)
Comment (Rey-2903) [reply .vscode/launch.json]: Deleted this file

**Review by zavarin-michael — APPROVED** [review]

**Pull request merged**